elephant/causality/granger.py

```diff
@@ -0,0 +1,438 @@
+# -*- coding: utf-8 -*-
+"""
+.. include:: causality-overview.rst
+
+.. current_module elephant.causality
+
+Overview of Functions
+---------------------
+Various formulations of Granger causality have been developed. In this module
+you will find function for time-series data to test pairwise Granger causality
+(`pairwise_granger`).
+
+Time-series Granger causality
+~~~~~~~~~~~~~~~
+
+.. autosummary::
+    :toctree: toctree/causality/
+
+    pairwise_granger
+
+Information Criterion
+~~~~~~~~~~~~~~~
+
+.. autosummary::
+    :toctree: toctree/causality/
+
+    bic
```

Comment:
I am wondering if `aic` and `bic` are only important for granger, or also for other causality functions (or even far beyond the causality module). My feeling is that they would at some point become part of a larger utility module.

Reply:
I think this might require quite a bit of discussion and planning.